heroic-component/src/main/java/com/spotify/heroic/statistics/MetricBackendReporter.java

```diff
@@ -33,4 +33,8 @@
     FutureReporter.Context reportQueryMetrics();
 
     void reportWritesDroppedBySize();
+
+    void reportGlobalReadDataPoints(long points);
```

Comment:
Not sure about `Global` as to the uninitiated it sounds like it's all clusters i.e. geographically Global. How about `All` or `Total`?

Reply:
Also where's the change to a concurrent map? Can't find it.

Reply:
Names are changed.
I've changed the implementation and using Collection.removeIf method that was suggested to use in this case.